app/assets/stylesheets/admin/questions/_questions.scss

```diff
@@ -78,3 +78,16 @@ section.questionnaires li.jqtree_common .jqtree-element
 .linked-document .help-block {
   margin: .25em 0;
 }
+
+.answer-wrapper.tree-view .decorated-numeric-input.form-element {
+  align-items: baseline;
+  display: inline-flex;
+
+  span.input-prefix {
+    margin-right: 0.5em;
+  }
+
+  span.input-postfix {
+    margin-left: 0.5em;
+  }
+}
```

Comment:
tried one class with .5em margin on each side, but having a left margin on prefix makes an indentation that doesn't look right.